index.js

```diff
@@ -68,7 +68,11 @@ function formatWithCursor(text, opts, addAlignmentSize) {
     return { formatted: text };
   }
 
-  text = stripBom(text);
+  const UTF8BOM = 0xfeff;
+  const isStripBom = text.charCodeAt(0) === UTF8BOM;
+  if (isStripBom) {
```

Comment:
`isStripBom` should probably be named `hasUnicodeBOM` instead

Reply:
agree, is better. done